### PR TITLE
Add Search Icon to search Navbar

### LINKED ISSUE #31: Add Magnifying glass to Search
- Add open source magnifying glass icon.

### PR DESCRIPTION
**Pull request by MauroGuerrero20**
## Features

- Adds Search Icon

image

### CODE REVIEW

**Review by antoniolinhart — APPROVED** [review]
LGTM! Nice change

**Review by VivekShankar — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ class Navbar extends Component {
           <ul className="navbar-nav">
             <form className="form-inline my-2 my-lg-0" onSubmit={this.handleSearchInput}>
               <input className="form-control mr-sm-2" id="search" type="search" placeholder="Search" aria-label="Search" />
-              <button className="btn btn-outline my-2 my-sm-0" type="submit"></button>
+              <button type="submit" style={{backgroundColor: "transparent", border: "none"}}><img src="/images/search_icon.png" alt="Search Icon"/></button>
```
Comment (steven-solar) [inline capstone-frontend/src/components/Navbar.jsx:61]: It might be nice to put the styles in a .css file to "separate concerns."

**Review by steven-solar — APPROVED** [review]
Looks great!

**Pull request merged**